src/config.rs

```diff
@@ -463,6 +465,27 @@ impl HtmlConfig {
     }
 }
 
+/// Configuration for how to fold chapters of sidebar.
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(default, rename_all = "kebab-case")]
+pub struct Fold {
+    /// When off, all folds are open. Default: `false`.
+    pub enable: bool,
+    /// The higher the more folded regions are open. When level is 0, all folds
+    /// are closed.
+    /// Default: `0`.
+    pub level: u8,
+}
+
+impl Default for Fold {
```

Comment:
Tiny tidbit: since the implementation just uses the default values of both properties you can just derive it instead of implementing it manually

Reply:
fixed 686d325